crowbar_framework/app/models/service_object.rb

```diff
@@ -822,8 +805,17 @@ def apply_role(role, inst, in_queue)
     # We also check that all nodes we'll require are in the ready state.
     #
 
-    # Initialize variables used in ensure at the end of the method
-    chef_daemon_nodes = []
+    nodes_without_admin = []
+    NodeObject.find_all_nodes.each do |node|
+      unless node[:platform] == "windows" || node.admin?
+        nodes_without_admin << node.name
+      end
+    end
+    apply_lock = Crowbar::Lock.new(
+      logger: @logger,
+      path: "/var/chef/cache/daemon-pause-file.lock",
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.